### PR TITLE
Refine the layout of basic-features.md

### PR DESCRIPTION
**Pull request by TomShawn**
### What is changed, added or deleted? (Required)
+ Refine the layout of basic-features.md
+ Correct the support status for TiDB Binlog

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v5.3 (TiDB 5.3 versions)
- [x] v5.2 (TiDB 5.2 versions)
- [x] v5.1 (TiDB 5.1 versions)
- [x] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s): https://github.com/pingcap/docs-cn/pull/7365 https://github.com/pingcap/docs/pull/6130

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by qiancai — APPROVED** [review]

**Comment by TomShawn** [comment]
@morgo Could you confirm the following two items? Thanks!

+ This issue has been closed. Can we remove this annotation? ```[^1]: TiDB incorrectly treats latin1 as a subset of utf8. See [TiDB #18955](https://github.com/pingcap/tidb/issues/18955) for more details.```
+ ```/sql-statements/sql-statement-select.md``` only lists the syntax supports related to `SELECT`, but it does not has a list of all SQL statement supported in TiDB. Can we remove this note? ```[^2]: See [Statement Reference](/sql-statements/sql-statement-select.md) for a full list of SQL statements supported.```

**Review by morgo — APPROVED** [review]

File: basic-features.md [hunk 1]
```diff
@@ -38,12 +44,16 @@ The following table provides an overview of the feature development history of T
...
+|----------------------------------------------------------------------------------------------------------|:------------:|:------------:|:------------:|:------------:|
 | Basic `SELECT`, `INSERT`, `UPDATE`, `DELETE`, `REPLACE`                                                  | Y            | Y            | Y            | Y            |
 | `INSERT ON DUPLICATE KEY UPDATE`                                                                         | Y            | Y            | Y            | Y            |
 | `LOAD DATA INFILE`                                                                                       | Y            | Y            | Y            | Y            |
 | `SELECT INTO OUTFILE`                                                                                    | Y            | Y            | Y            | Y            |
-| `INNER JOIN`, `LEFT\|RIGHT [OUTER] JOIN`                                                                 | Y            | Y            | Y            | Y            |
+| `INNER JOIN`, `LEFT|RIGHT [OUTER] JOIN`                                                                 | Y            | Y            | Y            | Y            |
```
Comment (morgo) [inline basic-features.md:56]: This is causing display issues in GitHub markdown. Hopefully it is not an issue in the live markdown.

Comment (TomShawn) [inline basic-features.md:56]:
```suggestion
| `INNER JOIN`, `LEFT\|RIGHT [OUTER] JOIN`                                                                 | Y            | Y            | Y            | Y            |
```

Comment (TomShawn) [reply basic-features.md]: Thanks! I change it back.

**Pull request merged**